bodhi/tests/server/services/test_updates.py

```diff
@@ -517,9 +518,21 @@ def test_provenpackager_request_privs(self, publish, *args):
                          csrf_token=app.get('/csrf').json_body['csrf_token'])
         res = app.post_json('/updates/%s/request' % str(nvr), post_data, status=400)
 
-        # Ensure we can't push it until it meets the requirements
+        # Ensure we can't push it until it passed CI
         eq_(res.json_body['status'], 'error')
-        eq_(res.json_body['errors'][0]['description'], config.get('not_yet_tested_msg'))
+        eq_(res.json_body['errors'][0]['description'],
```

Comment:
Since I'm just reformatting an existing assertion, I think I would prefer leaving this for another PR